lib/zipkin-tracer/zipkin_http_sender.rb

```diff
@@ -32,13 +32,18 @@ class ZipkinHttpSender < ZipkinSenderBase
     IP_FORMAT = :string
 
     def initialize(options)
-      SuckerPunch.logger = options[:logger]
       @json_api_host = options[:json_api_host]
+      @async = options[:async] != false
+      SuckerPunch.logger = options[:logger]
       super(options)
     end
 
     def flush!
-      AsyncHttpApiClient.perform_async(@json_api_host, spans.dup)
+      if @async
+        HttpApiClient.perform_async(@json_api_host, spans.dup)
+      else
+        HttpApiClient.new.perform(@json_api_host, spans.dup)
```

Comment:
I think we do not need to `.dup` here. Same comment for the other sender.
The reason we dup in async is to avoid the thread variable to be overwritten as the thread gets a new request while async is still to sent the spans of the previous request. In the sync case, that should not be an issue.

Reply:
done in f60e276